src/main/java/org/zendesk/client/v2/Zendesk.java

```diff
@@ -267,15 +267,24 @@ public Ticket getTicket(long id) {
                 handleList(User.class, "users")));
     }
 
-    public void deleteTicket(Ticket ticket) {
```

Comment:
might want to keep the old signature so it's backwards compatible?

Reply:
thank you, i updated it, it was a copy/paste removal